docs/customize/vocabularies/users.md

```diff
@@ -71,15 +71,26 @@ current_datastore.activate_user(user)
 db.session.commit()
 ```
 
-## Confirm user
+## Adding users and roles
 
-Only confirmed accounts can be logged in. There are two ways to confirm an account as an administrator:
+You may need to add users or make permission changes after the users vocabulary has been loaded. To create an account programmatically, run:
 
-When creating the account pass the `--confirm` parameter like so:
 ```shell
-pipenv run invenio users create email@domain.edu --confirm
+pipenv run invenio users create email@domain.edu --active --confirm
 ```
 
+This will automatically confirm the account. If you want to force the user to verify their email address, leave off the `--confirm` paramter.
```

Comment:
```suggestion
This will automatically confirm the account. If you want to force the user to verify their email address, leave off the `--confirm` parameter.
```